Makefile

```
@@ -69,22 +97,14 @@ install:
 	$(GO_CMD) install -v $(LDFLAGS) ./cmd/...
 
 image: yamls
-	$(IMAGE_BUILD_CMD) --build-arg VERSION=$(VERSION) \
-	    --target full \
-	    --build-arg HOSTMOUNT_PREFIX=$(CONTAINER_HOSTMOUNT_PREFIX) \
-	    --build-arg BASE_IMAGE_FULL=$(BASE_IMAGE_FULL) \
-	    --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL) \
-	    -t $(IMAGE_TAG) \
-	    $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)) \
-	    $(IMAGE_BUILD_EXTRA_OPTS) ./
-	$(IMAGE_BUILD_CMD) --build-arg VERSION=$(VERSION) \
-	    --target minimal \
-	    --build-arg HOSTMOUNT_PREFIX=$(CONTAINER_HOSTMOUNT_PREFIX) \
-	    --build-arg BASE_IMAGE_FULL=$(BASE_IMAGE_FULL) \
-	    --build-arg BASE_IMAGE_MINIMAL=$(BASE_IMAGE_MINIMAL) \
-	    -t $(IMAGE_TAG)-minimal \
-	    $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)-minimal) \
-	    $(IMAGE_BUILD_EXTRA_OPTS) ./
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_FULL)
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_MINIMAL)
+
+image-all: IMAGE_BUILD_CMD=$(IMAGE_BUILDX_CMD) # --load : not implemented yet see: https://github.com/docker/buildx/issues/59
+image-all: ensure-buildx yamls
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_FULL)
+	$(IMAGE_BUILD_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_MINIMAL)
```

Wouldn't this simplify a bit, still(?)

```suggestion
image-all: ensure-buildx yamls
	# --load : not implemented yet see: https://github.com/docker/buildx/issues/59
	$(IMAGE_BUILDX_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_FULL)
	$(IMAGE_BUILDX_CMD) $(IMAGE_BUILD_ARGS) $(IMAGE_BUILD_ARGS_MINIMAL)
```